.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -36,9 +36,14 @@ Note: remove the backticks (``) from the link.
 - [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
 
 <!--
+If this release will become the next LTS release, add/update the checkboxes above as needed e.g.
```

Comment:
I can see the release engineer not knowing whether it is an LTS candidate. Instructions here may be helpful.

Reply:
It's not yet determined how the LTS weeks will be presented (likely via something like RSS or an infrequently-updated calendar/table), but when that is in place we'll add a link here.

Reply:
Agree with the idea to link to instructions for determining if a release must go to LTS.

Reply:
Will address in #26256